plugin/src/App/StatusPages/NotConnected.lua

```diff
@@ -90,6 +88,8 @@ function NotConnectedPage:render()
 		}),
 
 		AddressEntry = e(AddressEntry, {
+			hostText = self.props.hostText,
+			portText = self.props.portText,
```

Comment:
This may cause the current value the user has typed in to be wiped out if Roact triggers a re-render here. We might want to switch this input to have a controlled value instead of using a ref.